File: sistema veterinario/Ave.java
public class Ave extends Animal
{
    public Ave(String nombre,
               int edad,
               double peso,
               Propietario propietario)
    {
        super(nombre, edad, peso, propietario);
    }

    @Override
    public void aplicarTratamiento()
    {
        System.out.println("Aplicando tratamiento para ave.");
    }
}

File: Logica/Ave.cs
Ave.cs                                                               namespace Sistema_Veterinario.Logica
{
    public class Ave : Animal
    {
        public Ave(string nombre,
                  int edad,
                  Propietario dueño)
            : base(nombre, edad, dueño)
        {
        }

        public override void AplicarTratamiento()
        {
            Console.WriteLine(
                "Tratamiento para aves");
        }
    }
}